src/components/TextField/OfficeTextField.vue

```diff
@@ -56,20 +64,51 @@
         @Model("input", {type: String}) private text!: string;
 
         @Prop({type: Boolean, default: false}) private disabled!: boolean;
+        @Prop({type: Boolean, default: false}) private autoAdjustHeight!: boolean;
         @Prop({type: Boolean, default: false}) private multiline!: boolean;
         @Prop({type: Object, default: null}) private iconProps!: IOfficeIconStyleProps;
         @Prop({type: Boolean, default: false}) private borderless!: boolean;
         @Prop({type: String, default: ""}) private inputClassName!: string;
         @Prop({type: String, default: null}) private iconClass!: string;
         @Prop({type: String, default: ""}) private label!: string;
+        @Prop({type: String, default: undefined}) private prefix!: string;
+        @Prop({type: String, default: undefined}) private suffix!: string;
         @Prop({type: String, default: null}) private errorMessage!: string;
         @Prop({type: Boolean, default: false}) private required!: boolean;
         @Prop({type: Boolean, default: false}) private resizable!: boolean;
         @Prop({type: Boolean, default: false}) private underlined!: boolean;
+        @Prop({
+            type: () => {
```

Comment:
Don't take callbacks as props. Just bubble the "focus" or "blur" event directly up: `@focus="$emit('focus', $event)`